sebaztyn/EPIC-Mail
# chore(user): Redesign homepage UI

#### What does this PR do?
- A redesign of the UI to fit the design pattern recommended

#### Description of Task to be completed?
- I re-worked the CSS stylesheets.
- I removed all previous colors used
- I took off the previous background image used.
- I re-indented the CSS stylesheets to 2 spaces
- I completely took off the write up in this design
- I took off centering from input elements

#### How should this be manually tested?
- A visit to https://sebaztyn.github.io/EPIC-Mail/UI/login.html should show a blue design as against the previous dark-pink design

#### What are the relevant pivotal tracker stories?
- PT Story link - https://www.pivotaltracker.com/story/show/164618387
